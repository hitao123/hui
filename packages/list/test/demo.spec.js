describe('list component', () => {
  it('should show list', () => {

  });
});
