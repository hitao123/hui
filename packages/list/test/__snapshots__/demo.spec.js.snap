// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div class="demo-list">
  <div class="h-list">
    <div class="h-list__loading" style="display:none;">
      <div class="h-loading h-loading--circular h-loading--black"><span class="h-loading__spinner h-loading__spinner--circular"> <svg viewBox="25 25 50 50" class="h-loading__circular"><circle cx="50" cy="50" r="20" fill="none"></circle></svg></span></div> <span class="h-list__text">加载中...</span>
    </div>
  </div>
</div>
`;
